src/main/java/org/akaza/openclinica/domain/enumsupport/EventCrfWorkflowEnum.java

```diff
@@ -0,0 +1,26 @@
+package org.akaza.openclinica.domain.enumsupport;
+
+import core.org.akaza.openclinica.i18n.util.ResourceBundleProvider;
+
+import java.util.HashMap;
+import java.util.ResourceBundle;
+
+/**
+ * The EventCrfWorkflowEnum enumeration.
+ */
+public enum EventCrfWorkflowEnum {
```

Comment:
can you rename this to EventCrfWorkflowStatusEnum since this is representing the status?

Reply:
done